MA-Project-Template.md

```diff
@@ -0,0 +1,55 @@
+# Template for a Modelica Association Project application #
+
+## Project name ##
+
+*[Name of the project]*
+
+## Project purpose ##
+
+*[Short description about the purpose of the project and previous work on
+  which the project builds (e.g. previous documents)]*
+
+## License of project results ##
+
+*[Results of the project must be publicly available under an open source license
+  approved by the Open Source Initiative, see <http://www.opensource.org/licenses>.
+  Define the license used in this project here.]*
+
+## Project rules ##
+
+*[Rules under which this project are carried out. Recommendations:*
+
+***Project meetings***
+
+*Project meetings are open to the public.*
+
+***Project members***
+
+  - *Project members are individual persons or organizations according to § 3 of the MA Bylaws.*
+
+  - *Every member has one vote.*
+
```

Comment:
Shouls we add here that every member needs to sign the MA CLA?

Reply:
Done in https://github.com/modelica/MA-Bylaws/pull/5/commits/4186c44e270af8cc94378bffac2eb1ff637d8efa